KomaMRIPlots/src/ui/DisplayFunctions.jl

```diff
@@ -154,60 +154,57 @@ julia> plot_seq(seq)
-	Gx =  reduce(vcat, [KomaMRICore.get_theo_A(seq.GR[1,i];off_val) for i=1:N])
-	Gy =  reduce(vcat, [KomaMRICore.get_theo_A(seq.GR[2,i];off_val) for i=1:N])
-	Gz =  reduce(vcat, [KomaMRICore.get_theo_A(seq.GR[3,i];off_val) for i=1:N])
-	#RFS
-	t2 =  reduce(vcat, [KomaMRICore.get_theo_t(seq.RF[1,i];max_rf_samples) .+ T0[i] for i=1:N])
-	R =   reduce(vcat, [KomaMRICore.get_theo_A(r;off_val,max_rf_samples) for r = seq.RF])
-	#ADC
-	t3 =  reduce(vcat, [KomaMRICore.get_theo_t(seq.ADC[i])  .+ T0[i] for i=1:N])
-	D =   reduce(vcat, [KomaMRICore.get_theo_A(d;off_val) for d = seq.ADC])
-    #Plot
-	p = [scatter() for j=1:(3+2O+1)]
-	#GR
-	p[1] = scatter(x=t1x*1e3, y=Gx*1e3,name=idx[1],hovertemplate="(%{x:.4f} ms, %{y:.2f} mT/m)",
-			xaxis=xaxis,yaxis=yaxis,legendgroup="Gx",showlegend=showlegend,marker=attr(color="#636EFA"))
-	p[2] = scatter(x=t1y*1e3, y=Gy*1e3,name=idx[2],hovertemplate="(%{x:.4f} ms, %{y:.2f} mT/m)",
-			xaxis=xaxis,yaxis=yaxis,legendgroup="Gy",showlegend=showlegend,marker=attr(color="#EF553B"))
-	p[3] = scatter(x=t1z*1e3, y=Gz*1e3,name=idx[3],hovertemplate="(%{x:.4f} ms, %{y:.2f} mT/m)",
-			xaxis=xaxis,yaxis=yaxis,legendgroup="Gz",showlegend=showlegend,marker=attr(color="#00CC96"))
-	#RF
-	for j=1:O
-		phase = angle.(R[:,j])
-		phase[R[:,j] .== Inf] .= Inf
-		p[2j-1+3] = scatter(x=t2*1e3, y=abs.(R[:,j])*1e6,name="|B1|",hovertemplate="(%{x:.4f} ms, %{y:.2f} μT)",
-					xaxis=xaxis,yaxis=yaxis,legendgroup="|B1|",showlegend=showlegend,marker=attr(color="#AB63FA"))
-		p[2j+3] =   scatter(x=t2*1e3, y=phase, text=ones(size(t2)), name="<B1",hovertemplate="(%{x:.4f} ms, ∠B1: %{y:.4f} rad)", visible="legendonly",
-					xaxis=xaxis,yaxis=yaxis,legendgroup="<B1",showlegend=showlegend,marker=attr(color="#FFA15A"))
+
+    # Get the samples of the events in the sequence
+    off_val = Inf       # This removes the unnecessary points in the plot
+    samples = get_samples(seq; off_val, max_rf_samples)
+    t_gx, t_gy, t_gz = samples.t_gx, samples.t_gy, samples.t_gz
+    t_rf, t_adc = samples.t_rf, samples.t_adc
+    A_gx, A_gy, A_gz = samples.A_gx, samples.A_gy, samples.A_gz
+    A_rf, A_adc = samples.A_rf, samples.A_adc
+
+    # Define general params and the vector of plots
+    idx = ["Gx" "Gy" "Gz"]
+	O = size(seq.RF, 1)
+	p = [scatter() for _ in 1:(3 + 2O + 1)]
+
+    # For GRADs
+	p[1] = scatter(x=t_gx*1e3, y=A_gx*1e3, name=idx[1], hovertemplate="(%{x:.4f} ms, %{y:.2f} mT/m)",
+		    xaxis=xaxis, yaxis=yaxis, legendgroup="Gx", showlegend=showlegend, marker=attr(color="#636EFA"))
+	p[2] = scatter(x=t_gy*1e3, y=A_gy*1e3, name=idx[2], hovertemplate="(%{x:.4f} ms, %{y:.2f} mT/m)",
+			xaxis=xaxis, yaxis=yaxis, legendgroup="Gy", showlegend=showlegend, marker=attr(color="#EF553B"))
+	p[3] = scatter(x=t_gz*1e3, y=A_gz*1e3,name=idx[3], hovertemplate="(%{x:.4f} ms, %{y:.2f} mT/m)",
+			xaxis=xaxis, yaxis=yaxis, legendgroup="Gz", showlegend=showlegend, marker=attr(color="#00CC96"))
```

Comment:
```suggestion
	p[1] = scatter(x=samples.gx.t*1e3, y=samples.gx.A*1e3, name=idx[1], hovertemplate="(%{x:.4f} ms, %{y:.2f} mT/m)",
		    xaxis=xaxis, yaxis=yaxis, legendgroup="Gx", showlegend=showlegend, marker=attr(color="#636EFA"))
	p[2] = scatter(x=samples.gy.t*1e3, y=samples.gy.A*1e3, name=idx[2], hovertemplate="(%{x:.4f} ms, %{y:.2f} mT/m)",
			xaxis=xaxis, yaxis=yaxis, legendgroup="Gy", showlegend=showlegend, marker=attr(color="#EF553B"))
	p[3] = scatter(x=samples.gz.t*1e3, y=samples.gz.A*1e3,name=idx[3], hovertemplate="(%{x:.4f} ms, %{y:.2f} mT/m)",
			xaxis=xaxis, yaxis=yaxis, legendgroup="Gz", showlegend=showlegend, marker=attr(color="#00CC96"))
```

Reply:
Addressed in https://github.com/cncastillo/KomaMRI.jl/pull/240/commits/a7bff9354c965bc1cd07216ff427da9d84ee75ee